documentation/source/user_section/tutorials/registration-to-template/lumbar-registration/before-starting.rst

```diff
@@ -0,0 +1,15 @@
+Before starting this tutorial
+#############################
+
+#. Make sure that you have the following files in your working directory:
+
+   * ``single_subject/data/t2_lumbar/t2.nii.gz`` : An anatomical spinal cord scan in the T2 contrast showing visible lumbar regions.
```

Comment:
I'm starting to question the idea of using separate subfolders when dealing with `t2_lumbar` and `t2_compression` tutorials. If these tutorials get integrated into the SCT course, for example, it's a bit awkward to have to `cd` from `t2/` into these other folders and then back again... :thinking: 

I'd like to test and see how the flow of the tutorials works in practice? (Are we even including lumbar/compression tutorials in the course?)